pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -465,6 +465,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
 				authorizationapi.NewRule("list", "watch").Groups(kapiGroup).Resources("endpoints").RuleOrDie(),
+				authorizationapi.NewRule("list", "watch").Groups(kapiGroup).Resources("services").RuleOrDie(),
```

Comment:
The router has to be able to get the Service IP, for when the route is idled.  The router doesn't actually send the unidling event -- it just forwards to the service proxy when it sees something that's idled.